lib/api/sfApi.js

```diff
@@ -123,25 +123,44 @@ async function readReconciliationTextById(firmId, id, refreshToken = true) {
   }
 }
 
+/**
+ * Find reconciliation by handle. We only look for templates that are not from Partners (because tecnically they don't belong to the current firm). If we find a template from Partners we will skip it and continue looking for the next one (while we keep it's `marketplace_template_id` as a reference)
+ * @param {Number} firmId
+ * @param {String} handle
+ * @param {Number} page
+ * @returns {Object} Reconciliation Text object
+ */
 async function findReconciliationTextByHandle(firmId, handle, page = 1) {
+  let marketplace_template_id;
   const reconciliations = await readReconciliationTexts(firmId, page);
-  // No data
+  // No data. Not found
   if (reconciliations.length == 0) {
     console.log(`Reconciliation ${handle} not found`);
```

Comment:
This line seems to be never reached

Reply:
It is reach when we call a page with no more reconciliations in it. Basically, when the handle provided doesn't exists

Reply:
ah ok, I see in that case it is reached, I was testing the case where you're trying to import a marketplace recon, so it does exist, but it also doesn't import anything. So it should also give a message that it wasn't imported, but it silently doesn't do anything. 

For example this one: 
`bin/cli.js import-reconciliation --yes -f 542 -h vol_6_19`

Reply:
As discussed, I have removed `marketplace_template_id`. We will consider the best flow while developing the Partners API